cmd/moby/build.go

```diff
@@ -156,20 +185,24 @@ func enforceContentTrust(fullImageName string, config *TrustConfig) bool {
 
 // Perform the actual build process
 // TODO return error not panic
-func buildInternal(m Moby, pull bool) []byte {
+func buildInternal(m Moby, pull bool, silent bool) []byte {
```

Comment:
its a bit annoying, but one build has to do another, but we dont want to see the second one...